### PR TITLE
Add config file for circleci

### PR DESCRIPTION
**Pull request by mumarkhan999**
#### Story Link
[https://edlyio.atlassian.net/browse/EDS-123](https://edlyio.atlassian.net/browse/EDS-123)

#### PR Description

This PR contains the file changes related to add circle ci integration in this Repo.
#### Type of change

Please select the options that are relevant.

- [ ] Bug fix (non-breaking change which fixes an issue)
- [X] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] Documentation Change

#### Checklist before merging:

- [ ] Squased
- [ ] Reviewd

### CODE REVIEW

**Review by HamzaIbnFarooq — APPROVED** [review]
LGTM

**Review by imhassantariq — APPROVED** [review]

File: openedx_caliper_tracking/tests/test_caliper_kafka.py [hunk 1]
```diff
@@ -330,7 +330,7 @@ def test_send_system_recovery_email_with_success(self, send_notification_mock, l
     )
     @override_settings(
         CALIPER_KAFKA_SETTINGS={
-            'ERROR_REPORT_EMAIL': 'dummy@example.com',
+            'ERROR_REPORT_EMAIL': 'dummy@example.commm',
```
Comment (danialmalik) [inline openedx_caliper_tracking/tests/test_caliper_kafka.py]: Why is this change necessary?
